app/src/androidTest/java/com/github/sdp/mediato/AuthenticationActivityTest.java

```diff
@@ -149,13 +198,37 @@ public void testLaunchingPostActivityThrowsWithNull() {
     }
 
     /**
-     * Test expected behavior of launching the post activity with logged in user
+     * Test expected behavior of launching the post activity with signing in user
+     *
+     * @throws InterruptedException: for thread.sleep
      */
     @Test
-    public void testLaunchingPostActivitySucceedsWithUser() {
+    public void testLaunchingPostActivitySucceedsWithUserSigningIn() throws InterruptedException {
         login();
         activity.launchPostActivity(user);
+        Thread.sleep(2000);
```

Comment:
I can also see that the parameter value to sleep here is different than in other parts of the code. Is there a reason for that ? If the Thread.sleep is necessary, you could store the values in constants so that it's clear how each is different.